src/GitVersion.Core/Core/Attributes/JsonPropertyPatternAttribute.cs

```diff
@@ -15,25 +19,29 @@ public JsonPropertyPatternAttribute(string pattern, PatternFormat format = Patte
     }
 
     /// <summary>
-    /// The pattern of the property.
+    /// The Regular Expression pattern of the property.
     /// </summary>
     public string Pattern { get; }
 
     /// <summary>
-    /// The format of the pattern.
+    /// The format of the pattern. The JSON schema specification only supports RegEx.
     /// </summary>
+    [Obsolete("Use JsonPropertyFormatAttribute instead.")]
```

Comment:
I would say to remove the property, as it was not included in any released version

Reply:
Should I label its removal as a breaking change?

Reply:
@BinToss these attributes were added in v6 version that is still worked on, so it's not a breaking change at all.

This is the commit it was added https://github.com/GitTools/GitVersion/commit/93b40c505b201ebe895cb7c747a309cb081dface